### ISSUE TITLE
Not able to log in

### ISSUE THREAD

**Issue by Catriks** [YunoHost-Apps/homeassistant_ynh#206]
I have Yunohost installed on my local PC, and Home Assistant installed on that. I can access Home Assistant and get the login screen, however I can not login with any user. I have tried to log in with Yunohost admin user, my normal user, and a test user for wich I speficically gave permission to Home Assistant, even though I have set HA to allow all users.

It is not clear to me if Yunohost login is supposed to be used with Home Assistant, If not then it is not clearly indicated anywhere, nor can I find how to make a new user for Home Assistant?

- Hardware: **HP Elitedesk 800 G1**
- YunoHost version: **11.1.16**
- I have access to my server: **SSH/webadmin/direct**
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: **No**
- Using, or trying to install package version/branch: **Latest from the Yunohost app installer**

Screenshot after failed login attempt
https://imgur.com/O4mUbW6

**Comment by ewilly** [comment]
It's not pure install as you try to login with an other auth provider. 
==> Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: **Yes**
If you want a link between Ha and yunohost users you need to keep the auth provider definied in [configuration.yaml lines 12 & 13](https://github.com/YunoHost-Apps/homeassistant_ynh/blob/master/conf/homeassistant_conf_files/configuration.yaml)

**Comment by Catriks** [comment]
> It's not pure install as you try to login with an other auth provider. ==> Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: **Yes** If you want a link between Ha and yunohost users you need to keep the auth provider definied in [configuration.yaml lines 12 & 13](https://github.com/YunoHost-Apps/homeassistant_ynh/blob/master/conf/homeassistant_conf_files/configuration.yaml)

Thank you. How could have I found this information on my own? Or how could I have made a separate user login for HA, since it is not asked anywhere?

I was able to find more info on configuring the auth providers here https://www.home-assistant.io/docs/authentication/providers/ but I'm still not able to figure out how to link HA and Yunohost users.

**Comment by ewilly** [comment]
Without touching auth provider, HA users are yunohost users allowed to acces the apps. Those users have to use the same credentials they use to login the ynh sso portal. You can't create users in HA, users are managed by yunohost.

**Comment by Catriks** [comment]
Uhh, I'm confused... Are you saying that to log in to Home Assistant, I need to use the same username and password that I use to log into Yunohost? Because that is what I have been trying to do. Isn't that what I said in my original post? 

But you did also say "If you want a link between Ha and yunohost users you need to keep the auth provider definied in [configuration.yaml lines 12 & 13](https://github.com/YunoHost-Apps/homeassistant_ynh/blob/master/conf/homeassistant_conf_files/configuration.yaml)" which sounds like you are implying that YNH and HA users are different?

**Comment by ewilly** [comment]
You manually change the auth provider in your configuration.yalm,  it's why you can't connect anymore with your yunohost user credential.

**Comment by Catriks** [comment]
> You manually change the auth provider in your configuration.yalm, it's why you can't connect anymore with your yunohost user credential.

What makes you think I've changed anything in any config file??? I have not, I only looked at the auth provider configuration page (that I linked before) because I thought you said I **need** to change something. I have not changed anything, it is a fresh install of both Yunohost and Home Assistant. I have never been able to log into Home Assistant. I have reinstalled the Home Assistant application many times but it never works, it always has the same issue.

**Comment by ewilly** [comment]
> What makes you think I've changed anything in any config file??? I have not,
When i see your screenshot it mention "Home Assistant Local" and not "Command Line Authentication". 
Can you share your configuration.yaml `/home/yunohost.app/homeassistant/configuration.yaml` ?

**Comment by Catriks** [comment]
I got it working now, here is what I did:

1. I went to take a copy of the configuration.yaml file, which I did do (and then lost the copy, unfortunately) but then I remembered I dont even have HA installed atm -> apparently reinstalling HA does not reset config files, which I had assumed it did.

2. I did `rm -r /home/yunohost.app/homeassistant`, then reinstalled HA from the app installer.

So it is possible there was some changes done, but they would have been done through Yunohost GUI while trying to get HA to work, since it did not work to begin with, or the changes were left over from a buggy install. But I am 99% sure I did not edit it from terminal/SSH because I would have not know what to edit.

The HA login interface does say now "Command Line Authentication" and logging in works. Is it possible that it previosly said "Home Assistant Local" because I used a doman called "test.local"? I did try switching the HA domain back and forth to that same test.local and my noho.st domain, but it makes no difference and logging in works as intented. 

Anyway, thank you for the help and I'm happy to have this working now.
